code/controllers/subsystem/mapping.dm

```diff
@@ -132,11 +132,20 @@ SUBSYSTEM_DEF(mapping)
 	while (space_levels_so_far < current_map.space_ruin_levels)
 		add_new_zlevel("Ruin Area [space_levels_so_far+1]", ZTRAITS_SPACE)
 		++space_levels_so_far
+
 	// Create empty space levels
 	while (space_levels_so_far < current_map.space_empty_levels + current_map.space_ruin_levels)
 		empty_space = add_new_zlevel("Empty Area [space_levels_so_far+1]", list(ZTRAIT_LINKAGE = CROSSLINKED))
 		++space_levels_so_far
 
+	if(current_map.wilderness_levels)
+		var/list/FailedZs = list()
+
+		LoadGroup(FailedZs, "Wilderness Area", current_map.wilderness_directory, current_map.maps_to_spawn, default_traits = ZTRAITS_WILDS, height_autosetup = FALSE)
+
+		if(LAZYLEN(FailedZs))
+			CRASH("Ice wilds failed to load!")
```

Comment:
this is blocked behind the define yes? is that ok?

Reply:
This is hidden if in LOWMEMORYMODE, yeah. I'm not sure there's any way to detect errors in loading without also loading

Reply:
I meant the like full load, since these are in theory part of the station. if you're ok with them being optional then it's fine

Reply:
Oh yeah I'm fine with it. It is pretty chunky. It's why we also refuse to generate lavaland in lowmemorymode so I feel like the precedent is there